docs/modules/ROOT/pages/usage.adoc

```diff
@@ -285,6 +285,63 @@ Hello, World!
 You have 1 arguments! First is YOLO!%
 ----
 
+== Running Jython (.py) (EXPERIMENTAL)
```

Comment:
I grok the incentive to have it here but this is not really jbang support for jython - its example of using the jython library.

Reply:
Totally agree. I'll change the heading and update the pull request

`== Running Jython scripts`